toqito/state_metrics/trace_distance.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 
 from toqito.matrix_props import is_density
-from toqito.state_metrics import trace_norm
+from toqito.matrix_props.trace_norm import trace_norm
```

Comment:
Similar comment here:

```suggestion
from toqito.matrix_props import trace_norm
```